spring-cloud-stream/src/main/java/org/springframework/cloud/stream/binding/BindableContextWrapper.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright 2015 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.stream.binding;
+
+import java.util.Map;
+
+import org.springframework.cloud.stream.annotation.EnableModule;
+import org.springframework.context.ConfigurableApplicationContext;
+
+/**
+ * Exposes a context as a {@link Bindable} component, allowing for embedding {@link EnableModule}
+ * units as child contexts into an application.
+ *
+ * @author Marius Bogoevici
+ */
+public class BindableContextWrapper implements Bindable {
```

Comment:
I don't think we need this class anymore. Given the bindings happen via `smart-lifecycle`, I believe the BindingUtils is enough for the bindings to happen. What do you think?